asistente_ladm_col/gui/supplies/dlg_missing_snc_supplies.py

```diff
@@ -0,0 +1,309 @@
+                        output['qgis:refactorfields_2:OMISIONES_TERRENO'],
+                        output['qgis:refactorfields_3:COMISIONES_MEJORAS'],
+                        output['qgis:refactorfields_4:OMISIONES_MEJORAS'],
+                        output['qgis:refactorfields_5:COMISIONES_UNID_PH'],
+                        output['qgis:refactorfields_6:OMISIONES_UNID_PH'],
+                        output['qgis:refactorfields_7:COMISIONES_MZ'],
+                        output['qgis:refactorfields_8:OMISIONES_MZ'],
+                        output['qgis:refactorfields_9:COMISIONES_VR'],
+                        output['qgis:refactorfields_10:OMISIONES_VR']],
+                    'OUTPUT': self.gpkg_path,
+                    'OVERWRITE': True,
+                    'SAVE_STYLES': True},
+                                               feedback=self.custom_feedback)
+        except QgsProcessingException as e:
+            if self.custom_feedback.isCanceled():
+                # The algorithm can throw errors even if canceled, so catch them and if we canceled, silent them.
+                return False, 'Package results canceled!'
+            else:
+                msg = "QgsProcessingException (Package layers): {}".format(str(e))
+                self.logger.critical(__name__, msg)
+                self.show_message(msg, Qgis.Critical)
+
+        root = QgsProject.instance().layerTreeRoot()
+        results_group = root.addGroup(QCoreApplication.translate("MissingSncSuppliesDialog", "Results missing supplies"))
+
+        for layer_path in output_geopackage['OUTPUT_LAYERS']:
+            layer = QgsVectorLayer(layer_path, layer_path.split('layername=')[1], 'ogr')
+            self.names_gpkg += '{} '.format(layer_path.split('layername=')[1])
+            if not layer.isValid():
+                return False, QCoreApplication.translate("MissingSncSuppliesDialog", "There were troubles loading {} layer.").format(layer_path.split('layername=')[1])
+            QgsProject.instance().addMapLayer(layer, False)
+            results_group.addLayer(layer)
+            LayerNode = root.findLayer(layer.id())
+            LayerNode.setCustomProperty("showFeatureCount", True)
+
+        return True, ''
+
+    def generate_excel_report(self):
+        gdal.VectorTranslate(self.xlsx_path,
+                             self.gpkg_path,
+                             options='-f XLSX {}'.format(self.names_gpkg.strip()))
+
+    def disable_widgets(self):
+        """
+        The base dialog has widgets that we don't need in this subclass. so hide them.
+        """
+        self.label_uni.setVisible(False)
+        self.txt_file_path_uni.setVisible(False)
+        self.btn_browse_file_uni.setVisible(False)
+
+    def validate_inputs(self):
+        """
+        The dialog has inputs that are necessary to model works. so this function validates than the file exists and has the correct extension.
```

Comment:
See comment in validate_inputs from dlg_missing_cobol_supplies.py.

Reply:
Changed

Reply:
False

Reply:
Solved in PR #373